wallets/cln/client.js

```diff
@@ -1 +1,39 @@
+import { fetchWithTimeout } from '@/lib/fetch'
+import { assertContentTypeJson, assertResponseOk } from '@/lib/url'
+
 export * from '@/wallets/cln'
+
+export async function testSendPayment ({ socket, runePay }, { logger, signal }) {
+  // TODO: check rune
```

Comment:
Missing TODO.

We usually do "integration tests" here that involve the network unlike the formik/yup validation that triggers no network requests.

So for example, you could try to use the rune in some way that wouldn't work if the rune didn't exist but since we don't actually want to pay something (this was planned in #1341 but I got blocked by an issue), you can't actually use the `pay` method. Maybe it helps to look at `testSendPayment` from existing wallets.